src/main.ts

```diff
@@ -72,7 +74,7 @@ export async function run(): Promise<void> {
     const noTraffic = core.getBooleanInput('no_traffic');
     const revTraffic = core.getInput('revision_traffic');
     const tagTraffic = core.getInput('tag_traffic');
-    const labels = parseKVString(core.getInput('labels'));
+    const labels = Object.assign({}, defaultLabels(), parseKVString(core.getInput('labels')));
```

Comment:
This will overwrite our default labels if there is a collision from the user input. I feel like we shouldn't allow this.

```javascript
const labels = Object.assign(
  {},
  parseKVString(core.getInput("labels")),
  defaultLabels()
);```

Reply:
Hmm - I did this intentionally so that the user _could_ override our labels if they wanted. I think the same is true for Cloud Build (there are defaults, but the user can override them).